go.mod

```diff
@@ -1,17 +1,14 @@
-module github.com/metal-stack/ipmi-catcher
+module github.com/metal-stack/bmc-catcher
 
 go 1.14
 
 require (
-	github.com/go-openapi/spec v0.19.8 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/metal-stack/metal-go v0.7.6
+	github.com/metal-stack/go-hal v0.1.12-0.20201001083047-90e1043cb849
+	github.com/metal-stack/metal-go v0.9.3-0.20201012065727-c724c2699a32
```

Comment:
This has to be pinned to a fixed version before merging.

Here are the relevant PRs:
- https://github.com/metal-stack/metal-api/pull/114
- https://github.com/metal-stack/go-hal/pull/10